client/app/services/vm-details/vm-details.component.js

```diff
@@ -35,9 +41,11 @@ function ComponentController($stateParams, VmsService, ServicesState, sprintf, l
           actionsInclude: true,
         },
       },
+      listActions: [],
     });
-    EventNotifications.info(__("The contents of this page is a function of the current users's group."));
     resolveData();
+    EventNotifications.info(__("The contents of this page is a function of the current users's group."));
```

Comment:
Agreed, not a big deal, was just wondering if there was a specific reason... OOOOO TIME FOR REBASE ANYWAYS :p

@chalettu